wp-content/themes/core/components/blocks/logos/logos.php

```diff
@@ -1,26 +1,32 @@
 <?php
 declare( strict_types=1 );
 
+use Tribe\Project\Templates\Components\blocks\logos\Logos_Block_Controller;
+
 /**
  * @var array $args Arguments passed to the template
  */
 // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
-$c = Tribe\Project\Templates\Components\blocks\logos\Logos_Block_Controller::factory( $args );
+$c = Logos_Block_Controller::factory( $args );
 
 if ( empty( $c->logos ) ) {
 	return;
 }
 ?>
 
-<section <?php echo $c->classes(); ?> <?php echo $c->attributes(); ?>>
-	<div class="l-container b-logos__container">
+<section <?php echo $c->classes(); ?><?php echo $c->attributes(); ?>>
+	<div <?php echo $c->container_classes(); ?><?php echo $c->container_attrs(); ?>>
```

Comment:
I don't need the extra args for: `container_classes`, `container_attrs`, `content_classes`